x-pack/plugins/observability_solution/observability_ai_assistant_app/scripts/evaluation/evaluation.ts

```diff
@@ -208,6 +208,15 @@ function runEvaluations() {
                 score.reasoning,
               ]);
             });
+
+            if (result.errors.length) {
+              output.push(['Errors', '', '']);
```

Comment:
This doesn't read well.

Reply:
is the issue about the tuple? or is the meaning of `result.errors.length` unclear?

Reply:
It's the tuple. An object with named properties would make it possible to see what the empty strings mean. 
Should have added this as a nit though.

Reply:
yeah I'm just using the `table` API directly. agree that it's not very readable but it's probably also not worth it to write an abstraction on top of that.